open-metadata-resources/open-metadata-labs/egeria-server-config.ipynb

```diff
@@ -1154,6 +1162,155 @@
+    "#  metadataServerUserName - user name with access asset-lineage omas\n",
+    "#  metadataServerUserPassword - password for the user with access to asset-lineage omas\n",
+    "#  inTopicName - suffix to the topic name configured on the server. When combined with topicRootURL should match the asset-lineage out Topic name.\n",
+    "\n",
+    "\n",
+    "def configureOpenLineageService(adminPlatformURL, adminUserId, lineageServerName, metadataPlatformURL, metadataServerName, metadataServerUserName, metadataServerUserPassword, inTopicName):\n",
+    "    adminCommandURLRoot = adminPlatformURL + '/open-metadata/admin-services/users/' + adminUserId + '/servers/'\n",
+    "    print (\"   ... configuring the Open Lineage Services server \" + lineageServerName + \" ...\")\n",
+    "    jsonContentHeader = {'content-type':'application/json'}\n",
+    "    requestBody =  {\n",
+    "       \"class\": \"OpenLineageConfig\",\n",
+    "       \"openLineageDescription\": \"Open Lineage Service is used for the storage and querying of lineage\",\n",
+    "       \"inTopicName\": inTopicName,\n",
+    "       \"lineageGraphConnection\": {\n",
+    "         \"class\": \"Connection\",\n",
+    "         \"displayName\": \"Lineage Graph Connection\",\n",
+    "         \"description\": \"Used for storing lineage in the Open Metadata format\",\n",
+    "         \"connectorType\": {\n",
+    "           \"class\": \"ConnectorType\",\n",
+    "           \"connectorProviderClassName\": \"org.odpi.openmetadata.openconnectors.governancedaemonconnectors.openlineageconnectors.janusconnector.graph.LineageGraphConnectorProvider\"\n",
+    "         },\n",
+    "         \"configurationProperties\": {\n",
+    "           \"gremlin.graph\": \"org.janusgraph.core.JanusGraphFactory\",\n",
+    "           \"storage.backend\": \"berkeleyje\",\n",
+    "           \"storage.directory\": \"data/servers/\"+ lineageServerName + \"/lineage-repository/berkeley\",\n",
+    "           \"index.search.backend\": \"lucene\",\n",
+    "           \"index.search.directory\": \"data/servers/\"+ lineageServerName + \"/lineage-repository/searchindex\"\n",
+    "         }\n",
+    "       },\n",
+    "       \"accessServiceConfig\": {\n",
+    "         \"serverName\": metadataServerName,\n",
+    "         \"serverPlatformUrlRoot\": metadataPlatformURL,\n",
+    "         \"user\": metadataServerUserName,\n",
+    "         \"password\": metadataServerUserPassword\n",
+    "       },\n",
+    "       \"backgroundJobs\": [\n",
+    "         {\n",
+    "           \"jobName\": \"LineageGraphJob\",\n",
+    "           \"jobInterval\": 120,\n",
+    "           \"jobEnabled\": \"false\"\n",
+    "         }, \n",
+    "         {\n",
+    "           \"jobName\": \"AssetLineageUpdateJob\",\n",
+    "           \"jobInterval\": 120,\n",
+    "           \"jobEnabled\": \"false\",\n",
+    "           \"jobDefaultValue\": \"2021-12-03T10:15:30\"\n",
```

Comment:
Is the future date of any consequence?

Reply:
Yes this is just random date. This part is disabled for now since it is not used. More details can be found on the pointer line above where full configuration is explained. Now, the plan is to add second exercise where this part will become relevant.

Reply:
ok